bootloader/bootloadertest/bootloadertest.go

```diff
@@ -474,29 +461,29 @@ func (b *MockTrustedAssetsBootloader) BootChain(runBl bootloader.Bootloader, ker
 	return b.BootChainList, b.BootChainErr
 }
 
-// MockManagedAssetsRecoveryAwareBootloader mocks a bootloader implementing the
-// bootloader.ManagedAssetsBootloader and bootloader.RecoveryAwareBootloader
+// MockTrustedAssetsRecoveryAwareBootloader mocks a bootloader implementing the
+// bootloader.TrustedAssetsBootloader and bootloader.RecoveryAwareBootloader
 // interfaces.
-type MockManagedAssetsRecoveryAwareBootloader struct {
-	*MockManagedAssetsBootloader
+type MockTrustedAssetsRecoveryAwareBootloader struct {
+	*MockTrustedAssetsBootloader
 
 	EnvVars map[string]string
 }
 
-func (b *MockBootloader) WithManagedAssetsRecoveryAware() *MockManagedAssetsRecoveryAwareBootloader {
-	return &MockManagedAssetsRecoveryAwareBootloader{
-		MockManagedAssetsBootloader: &MockManagedAssetsBootloader{
+func (b *MockBootloader) WithTrustedAssetsRecoveryAware() *MockTrustedAssetsRecoveryAwareBootloader {
```

Comment:
mmh, this doesn't seem to be used? I don't see the same rename anywhere else, will we need this?

Reply:
Quite likely this was to be used in one of the branches I had, but either it was not pushed or the code got refactored during reviews. With that in mind and having looked at the code again, RecoveryAwareBootloader.GetRecoverySystemEnv() is also not needed.

Reply:
I would drop this but not Get, it seems strange for symmetry not to have it

Reply:
Fair enough, I'll leave GetRecoverySystemEnv(), but drop the mocked bootloader.